src/Microsoft.AspNetCore.Hosting/Internal/ActivityTrackingOptions.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Hosting.Internal
+{
+    public class ActivityTrackingOptions
```

Comment:
The minimum concrete change is to simply make ActivityTrackingOptions be internal rather than public (but you could also rip out goo associated with passing it around.